examples/tensorflow/real_time.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/python3
+
+# Copyright (c) 2022 Raspberry Pi Ltd
+# Author: Alasdair Allan <alasdair@raspberrypi.com>
+# SPDX-License-Identifier: BSD-3-Clause
+
+# A TensorFlow Lite example for Picamera2 on Raspberry Pi OS Bullseye
+#
+# Install necessary dependences before starting,
```

Comment:
Extraneous comma at end of line

Reply:
it's not extraneous it's just a single run on sentence.